backup.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+# Will be name of directory in backet dd-mm-yyyy_hh-mm-ss
+    timestamp=$(date +%d-%m-%Y_%H-%M-%S)
```

Comment:
Btw, please follow [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) for dates & times. It is needed because lex sort and date sort will be same in this case.